coalib/collecting/Collectors.py

```diff
@@ -82,7 +82,9 @@ def collect_files(file_paths, log_printer, ignored_file_paths=None,
         collected_files, file_globs_with_files = [], []
 
     _warn_if_unused_glob(log_printer, file_paths, file_globs_with_files,
-                         "No files matching '{}' were found.")
+                         "No files matching '{}' were found. "
+                         'If this rule is not required, you can remove it '
+                         'from your .coafile to deactivate this warning.')
```

Comment:
Something along the line: "remove the rule 'files = ...' from your .coafile?
I'm not sure, if this would be more helpful, than the current iteration.

However, if I could add the currently executing section, the message would be much more useful.
This info is actually available at the function call-site ([Processing.py Line 358](https://github.com/coala/coala/blob/master/coalib/processes/Processing.py)), so a change to the signature of "collect_files" could make this info accessible.
The only other call sites are in test-cases, which would have to be adapted as well.

All in all, this would require some more changes, but I should be able to do it.